pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -482,6 +482,53 @@ func aclToRules(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) (types.IPTabl
 		aclRule5.Action = []string{"-j", chainName}
 		aclRule5.ActionChainName = chainName
 		rulesList = append(rulesList, aclRule5)
+	} else if aclArgs.NIType == types.NetworkInstanceTypeSwitch {
```

Comment:
What should happen if we setup an IP address for zedrouter when there is a bridge i.e., bridgeIP is set? 
Should the check on line 434 check the NIType instead of the bridgeIP?

Reply:
I'll replace the check ```aclArgs.BridgeIP != ""``` with ```aclArgs.NITType != types.NetworkInstanceTypeSwitch```. I'll also make corresponding else change required at line 485.